app/models/address.rb

```diff
@@ -0,0 +1,15 @@
+class Address < ApplicationRecord
+  belongs_to :addressable, polymorphic: true
+
+  def full_name
```

Comment:
I think, this methods should be moved to a decorator if they are only used in the views